src/ArgentPonyWarcraftClient/Models/Asset.cs

```diff
@@ -19,5 +19,11 @@ public class Asset
         /// </summary>
         [JsonPropertyName("value")]
         public Uri Value { get; set; }
+
+        /// <summary>
+        /// Gets the file data id for the asset.
+        /// </summary>
+        [JsonPropertyName("file_data_id")]
+        public long FileDataId { get; set; }
```

Comment:
Found this as a missing field on the `Asset` class while working with the Covenant media